### PR TITLE
[17.0][MIG] partner_stage_only_confirmed: Migration to 17.0

### PR DESCRIPTION
**Pull request by yankinmax**
Forward-port of this PR:
- [ ] https://github.com/OCA/partner-contact/pull/1402

Depends on:
- [ ] https://github.com/OCA/partner-contact/pull/1709

### CODE REVIEW

**Review by ivantodorovich — APPROVED** [review]

File: partner_stage_only_confirmed/pyproject.toml
Comment (SilvioC2C) [inline partner_stage_only_confirmed/pyproject.toml:1]: What is this file?

Comment (yankinmax) [reply partner_stage_only_confirmed/pyproject.toml:1]: IDK, it's something new generated with `pre-commit`

**Review by gurneyalex — APPROVED** [review]

**Comment by yankinmax** [comment]
@gurneyalex can we merge this PR?

**Review by Camille0907 — APPROVED** [review]

**Pull request merged**